### ISSUE TITLE
Issue 5: Entrenamiento del modelo CNN y análisis del aprendizaje

### ISSUE THREAD

**Issue by sharafayyoub** [sharafayyoub/Actividad-redes-concolucionales#6]
Descripción:
Compila y entrena el modelo, registrando las métricas de rendimiento para observar cómo mejora con las épocas. Además, representa gráficamente la evolución de la precisión (accuracy) y la pérdida (loss) tanto en el entrenamiento como en la validación.

Tareas:

Compilar el modelo:

model.compile(optimizer='adam',
              loss='categorical_crossentropy',
              metrics=['accuracy'])

Entrenar el modelo durante al menos 10 épocas, usando el 10% del set de entrenamiento como validación:

history = model.fit(x_train, y_train, epochs=10, validation_split=0.1)

Visualizar el aprendizaje:

Graficar history.history['accuracy'] y history.history['val_accuracy'].

Graficar history.history['loss'] y history.history['val_loss'].

Comentar si el modelo mejora con el tiempo y si hay signos de sobreajuste.

Criterio de éxito:
Gráficos de entrenamiento y validación correctamente generados y analizados.
